lib/fathead/react_native/output.txt

```diff
@@ -0,0 +1,1410 @@
+ActivityIndicator	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>Displays a circular loading indicator.</p></section>	https://facebook.github.io/react-native/releases/0.40/docs/activityindicator.html#
+ActivityIndicator animating	R	ActivityIndicator.animating										
+ActivityIndicator color	R	ActivityIndicator.color										
+ActivityIndicator hidesWhenStopped	R	ActivityIndicator.hidesWhenStopped										
+ActivityIndicator size	R	ActivityIndicator.size										
+ActivityIndicator.animating	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>Whether to show the indicator (true, the default) or hide it (false).</p><pre><code>animating bool</code></pre></section>	https://facebook.github.io/react-native/releases/0.40/docs/activityindicator.html#animating
+ActivityIndicator.color	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>The foreground color of the spinner (default is gray).</p><pre><code>color color</code></pre></section>	https://facebook.github.io/react-native/releases/0.40/docs/activityindicator.html#color
+ActivityIndicator.hidesWhenStopped	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>Whether the indicator should hide when not animating (true by default).</p><pre><code>hidesWhenStopped bool</code></pre></section>	https://facebook.github.io/react-native/releases/0.40/docs/activityindicator.html#hideswhenstopped
+ActivityIndicator.size	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>Size of the indicator (default is 'small'). Passing a number to the size prop is only supported on Android.</p><pre><code>size enum('small', 'large'), number</code></pre></section>	https://facebook.github.io/react-native/releases/0.40/docs/activityindicator.html#size
+AdSupportIOS	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>You can edit the content above on GitHub and send us a pull request!</p></section>	https://facebook.github.io/react-native/releases/0.40/docs/adsupportios.html#
+AdSupportIOS AdSupportIOS.getAdvertisingId	R	AdSupportIOS.AdSupportIOS.getAdvertisingId										
+AdSupportIOS AdSupportIOS.getAdvertisingTrackingEnabled	R	AdSupportIOS.AdSupportIOS.getAdvertisingTrackingEnabled										
+AdSupportIOS.AdSupportIOS.getAdvertisingId	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p></p><pre><code>getAdvertisingId(onSuccess, onFailure)</code></pre></section>	https://facebook.github.io/react-native/releases/0.40/docs/adsupportios.html#getadvertisingid
```

Comment:
@Hestben looks like several of these contain an empty paragraph: `<p></p>` -- Please remove those 👍